lessons/02-html-blocks/index.html

```diff
@@ -0,0 +1,22 @@
+<!Doctype html>
+<HTML>
+    <head>
+<title>
+    Photos of cats 
+</title>
+    </head>
+    <body>
+       <h1>
+        My Favourite Cats   
+       </h1> 
+       <P>
```

Comment:
All tags in HTML are going to be lowercase, this would need to be `<p>`

Reply:
Thanks for your comments!